lib/oli_web/plugs/registration_captcha.ex

```diff
@@ -59,39 +59,55 @@ defmodule Oli.Plugs.RegistrationCaptcha do
     end
   end
 
-  defp render_captcha_error(conn, purpose) do
+  defp render_captcha_error(conn, :register_author) do
```

Comment:
I refactored render_captcha_error to match on the reason instead of having a cond in the body because the `use_pow_config(:author)` part is not actually common to all the cases.